frontend/packages/console-app/src/components/nodes/node-dashboard/NodeAlerts.tsx

```diff
@@ -184,8 +188,8 @@ const NodeAlerts: React.FC = ({ children }) => {
             totalKey={NodeQueries.CPU_TOTAL}
             limitKey={NodeQueries.POD_RESOURCE_LIMIT_CPU}
             requestedKey={NodeQueries.POD_RESOURCE_REQUEST_CPU}
-            limitState={cpuLimit?.limit}
-            requestedState={cpuLimit?.requested}
+            limitState={cpuLimit?.limit ?? (cpuLimit ? LIMIT_STATE.WARN : LIMIT_STATE.OK)}
+            requestedState={cpuLimit?.requested ?? (cpuLimit ? LIMIT_STATE.WARN : LIMIT_STATE.OK)}
```

Comment:
going to defer to someone else as to whether these are sane fallbacks in this context